app-containers/podman/metadata.xml

```diff
@@ -1,39 +1,49 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
-		<email>zmedico@gentoo.org</email>
-		<name>Zac Medico</name>
-	</maintainer>
-	<longdescription lang="en">
-		Podman (the POD MANager) is a tool for managing containers
-		and images, volumes mounted into those containers, and pods
-		made from groups of containers. Podman is based on libpod,
-		a library for container lifecycle management that is also
-		contained in this repository. The libpod library provides
-		APIs for managing containers, pods, container images,
-		and volumes.
-	</longdescription>
-	<use>
-		<flag name="btrfs">
-			Enables dependencies for the "btrfs" graph driver, including
-			necessary kernel flags.
-		</flag>
-		<flag name="cgroup-hybrid">
-			Default to hybrid (legacy) cgroup hierarchy instead of unified (modern).
-		</flag>
-		<flag name="fuse">
-			Enables fuse dependencies (fuse-overlayfs is especially useful
-			for rootless mode).
-		</flag>
-		<flag name="init">
-			Enables catatonit dependency required for podman run --init.
-		</flag>
-		<flag name="rootless">
-			Enables dependencies for running in rootless mode.
-		</flag>
-	</use>
-	<upstream>
-		<remote-id type="github">containers/podman</remote-id>
-	</upstream>
+  <maintainer type="person">
```

Comment:
Looks as you switched from tabs to spaces for indentation. Please stick to the previous style - beside other aspects, it makes the diff cleaner.

In case you're using vim (or neovim) you could set e.g. 'noexpandtab' to disable tab-to-whitespace-conversion temporarily.

Reply:
I'm a switcher from Vim to Emacs. Thank you for being strict about indentation, made me file https://bugs.gentoo.org/914621

Reply:
Done